build.sh

```diff
@@ -111,6 +111,9 @@ else
 fi
 format_error_files=""
 
+# Ignore those PEP8 rules
+PEP8_IGNORES="E121,E123,E126,E226,E24,E704"
```

Comment:
Should the E24 be E24x? I don't see an E24 in the pep8 script's documentation.
Where did this particular choice of pep8 exceptions come from?